api-client/pangea_api/knex.py

```diff
@@ -3,6 +3,11 @@
 
 DEFAULT_ENDPOINT = 'https://pangea.gimmebio.com'
 
+def clean_url(url):
+    if url[-1] == '/':
+        url = url[:-1]
+    return url
```

Comment:
From the client perspective paths with a trailing '/' are correct because they will work. They trigger a failover in the server though which I didn't like